homeassistant/components/script/__init__.py

```diff
@@ -238,37 +238,36 @@ async def _async_process_config(hass, config, component) -> None:
     """
     entities = []
 
-    for config_key in extract_domain_configs(config, DOMAIN):
```

Comment:
All script config is already rewritten to be stored under the `script` domain here: https://github.com/home-assistant/core/blob/4c764fd7cbc502b32dfbc406f912d1c7375f2aa7/homeassistant/components/script/config.py#L121-L124